docs/pages/concept/web.md

```diff
@@ -0,0 +1,159 @@
+# Web Renderer
+
+## Overview
+
+Web rendering is an experimental feature that lets you render websites.
+Furthermore, you can place other components on the website. We refer to this process as embedding.
+
+Make sure you have a compositor version built with web renderer support. The web renderer introduces additional dependencies and significantly increases the size of the compositor binaries. To minimize that impact, we are supporting two versions of the compositor, one with web renderer support and one without it.
+
+:::tip
+You can view working example [here](https://github.com/membraneframework/video_compositor/blob/master/examples/web_view.rs)
+:::
+
+## Embedding components
+
+Embedding is a process of displaying child component frames on a website. You can define the child components in the `children` field of the web view. Web renderer embeds the frames into HTML elements defined on the website. Those HTML elements can be specified using the JavaScript function `register_inputs`.
+
+### `register_inputs` function
+
+ ```ts
+ function register_inputs(...element_ids: string[]): void
+ ```
+
+Maps the passed element IDs to component IDs provided in the `children` field. Each element ID represents the ID of an HTML element. The function should be called as soon as possible. If the compositor can't map the component ID to the element ID, the web renderer won't render that component. `register_inputs` can be called several times during runtime. Web renderer injects it into the website's window object as a native function.
+
+### Embedding methods
+
+There are 3 embedding methods available:
+
+- `chromium_embedding` - Frames produced by the child components are passed directly to the chromium instance and displayed on HTML canvas. With this embedding method [`register_inputs`](#register_inputs-function) expects IDs of existing canvas elements. Passing frames to the chromium instance introduces one more copy operation on each input frame, which may cause performance problems for a large number of inputs.
+- `native_embedding_over_content` - Renders frames produced by the child components on top of the website's content.
+- `native_embedding_under_content` - Renders frames produced by the child components below the website's content. The website needs to have a transparent background. Otherwise, it will cover the frames underneath it.
```

Comment:
```suggestion
- `chromium_embedding` - Frames produced by child components are passed directly to a chromium instance and displayed on an HTML canvas. With this embedding method [`register_inputs`](#register_inputs-function) expects IDs of existing canvas elements. Passing frames to the chromium instance introduces one more copy operation on each input frame, which may cause performance problems for a large number of inputs.
- `native_embedding_over_content` - Renders frames produced by child components on top of the website's content.
- `native_embedding_under_content` - Renders frames produced by child components below the website's content. The website needs to have a transparent background. Otherwise, it will cover the frames underneath it.
```